src/PrestaShopBundle/Service/Command/AbstractCommand.php

```diff
@@ -88,7 +88,7 @@ public function execute()
     /**
      * Add cache:clear to the execution.
      */
-    public function addCacheClear()
+    public function addCacheClear($all = true)
```

Comment:
A method `addCacheClear` which does not add `cache:clear` depending of this new parameter. That's quite disturbing. :thinking:

Reply:
cache:clear delete everything and so is totally redundant with the previous commands.

Reply:
we can't change the method definition, it's a BC break. More, adding a boolean flag is a bad practice and the function name is not consistent with what is done.

IMHO, you should add two new functions here:
* addCacheClearCommand()
* addDoctrineClearCommands()

And deprecate the old one, and replace all usages of the deprecated function from Core